experiments/azure/cyclecloud/cpu/experiment/README.md

```diff
@@ -87,125 +87,143 @@ rm singularity-ce*.tar.gz
 
 Batch script:
 
-```console
```

Comment:
Just so you don't think I'm a nut, I use "console" a lot (and often over bash) because I think the blue looks pretty on the GitHub dark mode. Yes, it's that dumb.

Reply:
Well I changed it to `bash` because that looks nicer than `console` in VSCode...